rust/lance/src/dataset.rs

```diff
@@ -1633,6 +1653,8 @@ mod tests {
     #[rstest]
     #[tokio::test]
     async fn test_write_manifest(#[values(false, true)] use_legacy_format: bool) {
+        use lance_table::feature_flags::FLAG_INVALID;
```

Comment:
Seems like you should change the `assert_eq!(manifest.writer_feature_flags, 0);` below to have the V2 flag if expected.

Reply:
Good catch.  Fixed.